src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/resource/ActivitiesResource.kt

```diff
@@ -68,4 +76,56 @@ class ActivitiesResource(private val activitiesService: ActivitiesService) {
     @RequestBody @Valid createActivityRequest: CreateActivityRequest
   ): CreateActivityResponse =
     activitiesService.createActivity(createActivityRequest)
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_ACTIVITIES')")
+  @PostMapping("/activities/offender-program-profile")
```

Comment:
Slightly awkward url, but keen to keep to Nomis style naming .. still open to other suggestions

Reply:
maybe add {courseActivityId}  ?

Reply:
ok